src/Microsoft.Windows.CsWin32.Package/Microsoft.Windows.CsWin32.Package.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+
+    <!-- Special properties for analyzer packages. -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+    <NoWarn>$(NoWarn);NU5128;NU5127</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NuSpecFile>Microsoft.Windows.CsWin32.nuspec</NuSpecFile>
+
+    <PackageId>Microsoft.Windows.CsWin32</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Windows.CsWin32\Roslyn38\Microsoft.Windows.CsWin32.Roslyn38.csproj" Private="true" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\Microsoft.Windows.CsWin32\Roslyn40\Microsoft.Windows.CsWin32.Roslyn40.csproj" Private="true" ReferenceOutputAssembly="false" />
```

Comment:
📝 The Roslyn38 and Roslyn40 subfolders seemed the easiest way to achieve all of:

1. Share version.json
2. Not move settings.schema.json
3. Not have 3 top-level folders for one multi-targeted assembly